src/gui/src/dbDescriptors.h

```diff
@@ -67,11 +77,15 @@ class DbInstDescriptor : public Descriptor
   void makePlacementStatusOptions(std::vector<EditorOption>& options) const;
   void makeOrientationOptions(std::vector<EditorOption>& options) const;
   bool setNewLocation(odb::dbInst* inst, std::any value, bool is_x) const;
+
+  ord::OpenRoad* openroad_;
```

Comment:
Don't keep the whole app when all you need is sta.

Reply:
I see what you mean, I think that's going to require several more changes since when this instance or any of QWidgets might not have dbSta made. If the gui is coming from gui::show it will be, but -gui those will not be built till after those modules are built. I'll mark this draft and open a separate PR to fix the WARN-33 and add the text to the readme for globbing.

Reply:
If you delay the creation of the descriptors to init they rest of the app should be initialized

Reply:
Yeah, I had a different solution that solves the problem and fixes another pet peeve of mine with the layer tooltips.